onnxruntime/core/graph/model.cc

```diff
@@ -260,9 +260,10 @@ static Status LoadModel(const T& file_path, std::shared_ptr<Model>& p_model, con
     if (status.Category() == common::SYSTEM) {
       switch (status.Code()) {
         case ENOENT:
-          return ORT_MAKE_STATUS(ONNXRUNTIME, NO_SUCHFILE, "Load model failed. File doesn't exist");
+          return ORT_MAKE_STATUS(ONNXRUNTIME, NO_SUCHFILE, "Load model", ToMBString(file_path),
```

Comment:
>"Load model" [](start = 59, length = 12)

nit: need a space after 'model'. Maybe also put quotes around the file path so that the error isn't confusing if that is empty. e.g. "Load model from '", ToMBString(file_path), "' failed...